filebeat/module/apache2/access/test/test.log-expected.json

```diff
@@ -1,77 +1,30 @@
 [
     {
         "@timestamp": "2016-12-26T14:16:29.000Z", 
-        "apache2.access.body_sent.bytes": "209", 
-        "apache2.access.http_version": "1.1", 
-        "apache2.access.method": "GET", 
-        "apache2.access.remote_ip": "::1", 
-        "apache2.access.response_code": "404", 
-        "apache2.access.url": "/favicon.ico", 
-        "apache2.access.user_name": "-", 
+        "file.path": "/Users/ruflin/Dev/gopath/src/github.com/elastic/beats/filebeat/module/apache2/access/test/test.log", 
         "fileset.module": "apache2", 
         "fileset.name": "access", 
+        "http.request.method": "GET", 
+        "http.response.body_sent.bytes": "209", 
```

Comment:
@webmat I suggest to put this under http here even though it's not in ECS

Reply:
A few things:

* It's true that people should be able to nest things freely under ECS fields. However, us doing so in these first conversions may confuse people, no?
* Plus, we'd be introducing a breaking change for a field that will change again in the future, once we settle on a final ECS name.
  * The way this field is named is not an easy sell like "let's just add it as is to ECS", for me. Because with web servers you can track at least 6 different sizes: request header, body or total size. Same thing for response. So here I would stick with the previous field name under `apache.access` until we've added this to ECS.